bitcoin/secp256k1_main.py

```diff
@@ -239,23 +236,17 @@ def multiply(s, pub, usehex, rawpub=True):
     of the scalar s.
     ('raw' options passed in)
     '''
-    newpub = secp256k1.PublicKey(pub, raw=rawpub, ctx=ctx)
-    #see note to "tweak_mul" function in podle.py
-    res = secp256k1._tweak_public(newpub,
-                                   secp256k1.lib.secp256k1_ec_pubkey_tweak_mul,
-                                   s)
-    return res.serialize()
+    res = coincurve.PublicKey(pub)
+    res.multiply(s, update=True)
```

Comment:
Yes you're right (in fact I think that second variable wasn't needed in the old version, but no matter). I'll have to double check what that donation address code is doing (I haven't looked at it in over a year), but that's independent of the PR.